test/suites/perf/scheduling_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"sigs.k8s.io/karpenter/pkg/test"
 )
 
-var replicas int = 100
+var replicas int = 10
```

Comment:
scaling this back down to account for some kwok scalabilty issues at the moment.